pkg/expapi/types.go

```diff
@@ -110,3 +110,18 @@ type HorizontalPodAutoscalerList struct {
 
 	Items []HorizontalPodAutoscaler `json:"items"`
 }
+
+type Hello struct {
+	api.TypeMeta   `json:",inline"`
+	api.ObjectMeta `json:"metadata,omitempty"`
+
+	Text  string `json:"text,omitempty"`
+	Text2 string `json:"test,omitempty"`
```

Comment:
It's a typo. The entire Hello/HelloList struct is only used for testing. I will remove this commit before merge.